ext/pom.xml

```diff
@@ -44,6 +44,7 @@
         <module>cdi</module>
         <module>entity-filtering</module>
         <module>metainf-services</module>
+	    <module>microprofile/config</module>
```

Comment:
just microprofile. Each sub module should be dealt with in the microprofile project. There would be at least 2 submodules

Reply:
correct, but we already have a PR #4086 which does this with its module, so it would be less conflicts when both PR's will be merged